pkg/customcost/pipelineservice_test.go

```diff
@@ -0,0 +1,192 @@
+package customcost
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/opencost/opencost/core/pkg/log"
+	"github.com/opencost/opencost/core/pkg/util/timeutil"
+	"github.com/opencost/opencost/core/pkg/version"
+)
+
+func TestPipelineService(t *testing.T) {
+	// establish temporary test assets dir
+
+	dir := t.TempDir()
+
+	err := os.MkdirAll(dir+"/config", 0777)
```

Comment:
For the future, its better to use
```go
import "path/filepath"

os.MkdirAll(filepath.Join(dir, "config"))
```

instead of string ops when constructing paths